docs/apk-generator.md

```diff
@@ -1,18 +1,18 @@
-## How apk is generated using the Android apk generator
+# How apk is generated using the Android apk generator
 
 ## **Getting the data from uploaded zip/url**
 
-![Apk Generator](images/androidgenerator.png)
+![Apk Generator](_static/images/androidgenerator.png)
```

Comment:
Yeah, so? What I'm saying is that if an image isn't referenced from an embedded html, you don't need to keep it in static. Is `images/androidgenerator.png` referenced somewhere from embedded html? If not you can keep the images where they are. I don't understand how will the images not render on github properly if you are referencing using the correct path irrespective of where you keep it. Am I missing something?